### DISCUSSION TITLE
We should fix/split semaphoreci

### DISCUSSION THREAD

**Discussion by ljmf00** [dlang/dmd#13388] [CI]
Looking at the latest commits, semaphoreci fails quite frequently, and apparently, it is due to timeout, so we should either fix it by replacing it with an alternative or split the current job pipeline.

**Comment by ibuclaw** [comment]
I think the CirrusCI tests have made the semaphore pipelines redundant anyway.

Its a shame really, semaphore was really good 5 years ago.

**Reply by ljmf00** [reply]
As far as I know, it is the only CI that does some tests with LDC, but I don't really know what tests are made there.

**Reply by ibuclaw** [reply]
[Example](https://semaphoreci.com/dlang/dmd-2/branches/pull-request-13395/builds/12).

The following Cirrus CI pipelines are pretty much identical.
- [DMD x86](https://cirrus-ci.com/task/6459614905499648)
- [DMD x86_64](https://cirrus-ci.com/task/5615189975367680)
- [GDC 9 x86_64](https://cirrus-ci.com/task/5641578254434304)
- [LDC x86_64](https://cirrus-ci.com/task/4515678347591680)

**Reply by ljmf00** [reply]
Oh ok, I never noticed that. So we can safely remove it.

**Comment by ibuclaw** [comment]
SemaphoreCI is no longer a required pipeline for PRs to be merged.
